lib/parse/data/parseEmbedData.spec.ts

```diff
@@ -20,29 +20,29 @@ describe('lib/parse/data', () => {
       items: [
         {
           guid: 'foo-bar',
-          link: '//foo.com/foo-bar',
+          link: 'http://foo.com/foo-bar',
           title: 'Foo Bar',
           categories: ['foo', 'bar'],
           enclosure: {
-            url: '//foo.com/foo-bar.mp3'
+            url: 'http://foo.com/foo-bar.mp3?_from=play.prx.org'
           },
           itunes: {
             subtitle: 'Foo to the bar.',
-            image: '//foo.com/foo-bar.png',
+            image: 'http://foo.com/foo-bar.png',
             season: '1'
           }
         },
         {
           guid: 'foo-baz',
-          link: '//foo.com/foo-baz',
+          link: 'http://foo.com/foo-baz',
           title: 'Foo Baz',
           categories: ['foo', 'baz'],
           enclosure: {
-            url: '//foo.com/foo-baz.mp3'
+            url: 'http://foo.com/foo-baz.mp3?_from=play.prx.org'
```

Comment:
So the enclosure url has already passed through `parseAudioData` at this point?  (I'm guessing you've changed all your mocks to more-accurately already have the `_from` param)

Reply:
Actually, the mocks don't need it. This is probably a remnant of a find-and-replace of `.mp3` that I didn't clean up properly.